node/network/availability-distribution/src/requester/fetch_task/mod.rs

```diff
@@ -251,20 +258,18 @@ impl RunningTask {
 		let mut bad_validators = Vec::new();
 		let mut succeeded = false;
 		let mut count: u32 = 0;
-		let mut _span = self
-			.span
-			.child("fetch-task")
-			.with_chunk_index(self.request.index.0)
-			.with_relay_parent(self.relay_parent);
+		let mut span = self.span.child("run-fetch-chunk-task").with_relay_parent(self.relay_parent);
 		// Try validators in reverse order:
 		while let Some(validator) = self.group.pop() {
-			let _try_span = _span.child("try");
 			// Report retries:
 			if count > 0 {
 				self.metrics.on_retry();
 			}
 			count += 1;
-
+			let _chunk_fetch_span = span
+				.child("fetch-chunk-request")
```

Comment:
this also covers storing the span.

Reply:
Will separate spans into:
- fetch
- recombine
- validate-and-store